genesyscloud/provider.go

```diff
@@ -122,6 +122,7 @@ func New(version string) func() *schema.Provider {
 				"genesyscloud_telephony_providers_edges_linebasesettings":  dataSourceLineBaseSettings(),
 				"genesyscloud_telephony_providers_edges_phonebasesettings": dataSourcePhoneBaseSettings(),
 				"genesyscloud_telephony_providers_edges_trunkbasesettings": dataSourceTrunkBaseSettings(),
+				"genesyscloud_telephony_providers_edges_did_pool":			dataSourceDidPool(),
```

Comment:
Formatting is off, you can use go format to format the file. https://go.dev/blog/gofmt